src/dune_cache/dune_cache.ml

```diff
@@ -282,12 +282,13 @@ module Cache = struct
     Io.write_file metadata_tmp_path metadata;
     let () =
       match Io.read_file metadata_path with
-        | contents -> if contents <> metadata then
```

Comment:
This hunk doesn't seem necessary

Reply:
Removed. It was due to formatting during rebase.